src/tools/tidy/src/edition.rs

```diff
@@ -24,7 +24,10 @@ pub fn check(path: &Path, bad: &mut bool) {
             }
 
             // Library crates are not yet ready to migrate to 2021.
-            if path.components().any(|c| c.as_os_str() == "library") {
+            // TOOD: Once all library crates are migrated to 2021 then simplify the check to always check for 2021
```

Comment:
There's a typo in TODO.

(Maybe just say 'Not all library crate are ..' in the existing comment? We're force to update this anyway when another library crate moves to 2021, so it's not like we can forget anything here.)